Broker include broken

I have some external project that uses Zeek's `FindBroker.cmake` to link against broker. With the current zeek master the build stopped working. Configure exists with the following error:
```
CMake Error at /usr/local/zeek/share/zeek/cmake/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Broker (missing: BROKER_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/local/zeek/share/zeek/cmake/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  my-proj/cmake/FindBroker.cmake:74 (find_package_handle_standard_args)
  my-proj/CMakeLists.txt:14 (find_package)
```
Comparing with installs of previous Zeek versions, I noticed `config.hh` missing in `include/broker`. Maybe it's related to https://github.com/zeek/broker/commit/3c64a311d802348f020eaf3eda3042d4d9cb4b1d.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -263,8 +263,6 @@ install(DIRECTORY libbroker/broker
         PATTERN "libbroker/broker/internal" EXCLUDE
         PATTERN "*.test.hh" EXCLUDE)
 
-# install(FILES ${CMAKE_CURRENT_BINARY_DIR}/include/broker/config.hh DESTINATION include/broker)
-
 if (NOT BROKER_EXTERNAL_SQLITE_TARGET)
   include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty)
   set_source_files_properties(3rdparty/sqlite3.c PROPERTIES COMPILE_FLAGS

--- a/libbroker/CMakeLists.txt
+++ b/libbroker/CMakeLists.txt
@@ -1,6 +1,9 @@
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/broker/config.hh.in
                ${CMAKE_CURRENT_BINARY_DIR}/broker/config.hh)
 
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/broker/config.hh
+        DESTINATION include/broker)
+
 set(BROKER_SRC
   # src/detail/core_recorder.cc
   # src/detail/generator_file_reader.cc

